engine/api/workflow/workflow_exporter.go

```diff
@@ -29,23 +29,18 @@ func Export(ctx context.Context, db gorp.SqlExecutor, cache cache.Store, proj sd
 
 	// If repo is from as-code do not export WorkflowSkipIfOnlyOneRepoWebhook
 	if wf.FromRepository != "" {
-		opts = append(opts, exportentities.WorkflowSkipIfOnlyOneRepoWebhook)
+		opts.SkipIfOnlyOneRepoWebhook = true
 	}
 
-	return exportWorkflow(ctx, *wf, f, w, opts...)
+	return exportWorkflow(ctx, *wf, f, w, opts)
 }
 
-func exportWorkflow(ctx context.Context, wf sdk.Workflow, f exportentities.Format, w io.Writer, opts ...exportentities.WorkflowOptions) (int, error) {
-	e, err := exportentities.NewWorkflow(ctx, wf, opts...)
+func exportWorkflow(ctx context.Context, wf sdk.Workflow, f exportentities.Format, w io.Writer, opts exportentities.Options) (int, error) {
```

Comment:
* why did you change the variadic options, it is the most idiomatic way to deal with options ?
* At least we could pass a pointer of options, and consider nil à the default options (instead of passing an empty exportentities.Options{})

here are a few links about this:
https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis
https://www.sohamkamani.com/blog/golang/options-pattern/
https://github.com/tmrts/go-patterns/blob/master/idiom/functional-options.md